io-utils/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "io-utils"
+version = "0.3.0"
+authors = ["The Veracruz Development Team"]
+edition = "2018"
+description = "Common IO-related code used by multiple Veracruz components."
+
+[features]
+nitro = ["serde_json/std", "nix", "byteorder"]
+
+[dependencies]
+sgx_tstd = { rev = "v1.1.2", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
```

Comment:
`io-utils` is for now exclusively nitro-specific. Why have `sgx_std` as an optional dependency?

Reply:
I actually deleted that on my rebased `runtime-manager-linux` this morning.